### DISCUSSION TITLE
Vote to replay map at score screen in MP.

### DISCUSSION THREAD

**Discussion by GOINGCRAZYWITHIT** [ppy/osu#24687] [Ideas]
After a map is completed in MP I think an option to vote to replay, only appearing front and center for everyone once there is at least one vote, would be a nice QoL addition. I also believe it beats out other comparable options for such a feature, like: adding the song to the top of the queue, having to manually edit your next song in queue to the last song you just played (if it was even yours).

**Comment by peppy** [comment]
This sounds like a cool feature to have for sure.
